gemrb/includes/ie_types.h

```diff
@@ -108,6 +109,11 @@ enum class ieStrRef : ieDword {
 using ieVariable = FixedSizeString<32, strnicmp>;
 using ResRef = FixedSizeString<8, strnicmp>;
 
+using StringParam = FixedSizeString<64, strnicmp>;
+static_assert(std::is_standard_layout<StringParam>::value, "Fixed Size String must be standard layout for use in unions");
```

Comment:
will you just move this assert to the `union` declaration instead? it applies to everything in the union, I was just lazy and asserted for one because they are all `FixedSizeString`

Reply:
So you want this line back to `GameScript.h`, and also for both the other types (`ResRef`, `ieVariable`)?

Reply:
you don't need to add the others, they are all the same template and the size doesn't matter for standard layout.